### PR TITLE
Rfc metadata integration test

### PR DESCRIPTION
**Pull request by diekhans**
Added a section on using version numbers to allow data to proceed part way through the pipeline.

### CODE REVIEW

File: rfcs/text/0000-metadata-integration-test.md [hunk 1]
```diff
@@ -73,6 +74,26 @@ Data with the new schema version should not be uploaded into the `integration` o
...
+
+#### Pros:
+* Easy to implement, especially for metadata developers.
+* Good fit for the serial nature of the pipeline.
+* Works very will for simple changes that can be done quickly
+
+#### Cons:
+* Require serial integration of metadata changes in version number order, requiring quick response to incompatibilities.
+* Complex, time consuming changes could stall unrelated metadata updates from being published.
+
```
Comment (mweiden) [inline rfcs/text/0000-metadata-integration-test.md:98]: I would add another (big) con here. Data that is filtered out will often have to be replayed manually through the system or from component to component.

File: rfcs/text/0000-metadata-integration-test.md [hunk 2]
```diff
@@ -48,6 +48,7 @@ Data with the new schema version should not be uploaded into the `integration` o
 
 * May introduce another deployment stage
 * Treats metadata schema integration differently from other DCP projects
+* Requires test data to that use the new schema features
 
 ## Alternative design
```
Comment (mweiden) [inline rfcs/text/0000-metadata-integration-test.md:54]: Please add a title to the first alternative design to differentiate the two. Perhaps `### All projects must PR before merging into integration environment`

**Review by mweiden — CHANGES_REQUESTED** [review]
Just two small tweaks. Otherwise LGTM 👍

File: rfcs/text/0000-metadata-integration-test.md (hunk 1 above)
Comment (diekhans) [reply rfcs/text/0000-metadata-integration-test.md:98]: My intention here is that data is just queued waiting for the next stage to support it.  Once supported (a commit of an updated version list), then the queued submission can be processed.  I will updated the text to reflect this

**Pull request merged**